spec/models/collection_spec.rb

```diff
@@ -3,7 +3,7 @@
 describe Collection, :type => :model do
   before do
     @user = FactoryGirl.create(:user)
-    @collection = Collection.new(title: "test collection").tap do |c|
+    @collection = Collection.new(id: 'mock-collection-with-members', title: "test collection") do |c|
```

Comment:
The ID is needed because it is used in the bytes method. If you don't supply an ID then the {!join ...} Solr statement fails at https://github.com/narogers/sufia/blob/collection-speed/sufia-models/app/models/concerns/sufia/collection_behavior.rb#L23